packages/nvidia_nat_fastmcp/src/nat/plugins/fastmcp/server/front_end_plugin_worker.py

```
@@ -0,0 +1,333 @@
+
+            verifier_kwargs = {
+                "introspection_url": server_auth.introspection_endpoint,
+                "client_id": server_auth.client_id,
+                "client_secret": (server_auth.client_secret.get_secret_value() if server_auth.client_secret else None),
+                "required_scopes": server_auth.scopes,
+            }
+            if server_auth.client_auth_method:
+                verifier_kwargs["client_auth_method"] = server_auth.client_auth_method
+            verifier = IntrospectionTokenVerifier(**verifier_kwargs)
+            host = self.front_end_config.host
+            if host in {"0.0.0.0", "::"}:
```

Ruff (0.14.14)

[error] Possible binding to all interfaces

(S104)